vector/v.in.ogr/geom.c

```diff
@@ -361,6 +362,22 @@ int geom(OGRGeometryH hGeomAny, struct Map_info *Map, int field, int cat,
         }
         Vect_line_prune(Points);
 
+        lastidx = Points->n_points - 1;
+        if (Points->x[0] != Points->x[lastidx] ||
+            Points->y[0] != Points->y[lastidx] ||
+            Points->z[0] != Points->z[lastidx]) {
+            if (mk_centr) {
+                /* do not clean polygons */
+                G_fatal_error(_("Found unclosed outer polygon ring, can be "
+                                "closed when cleaning polygons is enabled"));
```

Comment:
Small remark: from the message it is not clear how to enable "cleaning polygons". Is snapping (`snap` parameter) meant?

Reply:
Cleaning polygons is enabled by default and can be disabled with the `-c` flag: "Do not clean polygons (not recommended)". Thus maybe a more correct message would be "Found unclosed outer polygon ring, can be closed when cleaning polygons is not disabled"?